clang/test/Frontend/sycl-aspects-usage.cpp

```diff
@@ -0,0 +1,42 @@
+// Test checks Propagate Aspect Usage pass.
+
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -emit-llvm %s -o /dev/null 2>&1 | FileCheck %s --check-prefix CHECK-WARNINGS
```

Comment:
Can we not use `-verify `here?

Reply:
Unfortunately at the moment those diagnostics are emitted without source location attached to them and therefore I expect that `-verify` won't work. I've submitted #5877 to track further improvement of that diagnostic which allow us to use `-verify`